### PR TITLE
IDVA6 1362 Remove page flow - api integration

### PR DESCRIPTION
**Pull request by bmchugh-ch**
Link to Jira

https://companieshouse.atlassian.net/browse/IDVA6-1362
https://companieshouse.atlassian.net/browse/IDVA6-1199

Remove ACSP member flow now working with the ACSP user api.
An account owner will be presented with stop screen informing them that they cant remove themself if they are the only account owner on the ACSP.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by mmodzelewski-ch — CHANGES_REQUESTED** [review]

File: src/locales/cy/translation/stop-page-add-account-owner.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
[the remark this answers is not in the document]
Comment (bmchugh-ch) [reply src/locales/cy/translation/stop-page-add-account-owner.json:1]: sorted

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by mmodzelewski-ch — APPROVED** [review]

**Pull request merged**